blocks/components/editable/index.js

```diff
@@ -1,3 +1,75 @@
-export default function Editable( { value } ) {
-	return <p>{ value }</p>;
+export default class Editable extends wp.element.Component {
+	constructor() {
+		super( ...arguments );
+		this.onInit = this.onInit.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+		this.onChange = this.onChange.bind( this );
+		this.bindNode = this.bindNode.bind( this );
+	}
+
+	componentDidMount() {
+		this.initialize();
+	}
+
+	initialize() {
+		const config = {
+			target: this.node,
+			theme: false,
+			inline: true,
+			toolbar: false,
+			browser_spellcheck: true,
+			entity_encoding: 'raw',
+			setup: this.onSetup,
+			formats: {
+				strikethrough: { inline: 'del' }
+			}
+		};
+
+		tinymce.init( config );
+	}
+
+	onSetup( editor ) {
+		this.editor = editor;
+		editor.on( 'init', this.onInit );
+		editor.on( 'focusout', this.onChange );
```

Comment:
focusout will when focus is shifted internally within nested contenteditables. Use "blur" instead or possible 'change' since that is called on every undo level add including 'blur' since that adds an undo level. Might be to frequent though.